tests/tx/test_tips.py

```diff
@@ -0,0 +1,182 @@
+from hathor.transaction import Transaction
+from tests import unittest
+from tests.utils import (
+    add_blocks_unlock_reward,
+    add_new_block,
+    add_new_blocks,
+    add_new_double_spending,
+    add_new_transactions,
+)
+
+
+class BaseTipsTestCase(unittest.TestCase):
```

Comment:
These unit tests feel good to me. But I feel we should avoid accessing `_tx_tips_index` and use the public method instead. So we can easily extend them to other indexes in the future.

Reply:
Changing to use public methods only.

Reply:
Great! Thanks!